setup.cfg

```
@@ -22,7 +22,7 @@ install_requires =
     matplotlib
     traitlets>=5.0.5
     bqplot>=0.12.33
-    bqplot-image-gl>=1.4.7
+    bqplot-image-gl==1.4.7
```

Maybe more future-proof like this?
```suggestion
    bqplot-image-gl>=1.4.7,!=1.4.8
```

---

I'll let @rosteen decide...

---

He stepped out, is there any reason not to commit that?

---

OK, let's YOLO and see if CI passes.